### DISCUSSION TITLE
Loading CSV file

### DISCUSSION THREAD

**Discussion by Nambunga** [IBAHCM/RPiR#178] [Archive] [answered]
@soniamitchell @richardreeve While importing the CSV file to my R script for making a table in practical 2-2, I got a warning message saying  _read.table(file = file, header = header, sep = sep, quote = quote,  : incomplete final line found by readTableHeader on '0202_SIS_table.csv'_. Could you please help to solve this?

**Comment by soniamitchell** [comment, marked as answer]
Hey @Nambunga, did you get this sorted? I'd suggest using `read.csv()` to read css files instead of `read.table()`, see if that helps?

**Reply by Nambunga** [reply]
@soniamitchell Thanks, it's fine now
